pkg/providers/cloudstack/config/template-cp.yaml

```diff
@@ -497,6 +500,9 @@ spec:
 {{- end}}
 {{- if .cloudstackEtcdDiskOfferingName}}
         name: {{.cloudstackEtcdDiskOfferingName}}
+{{- end}}
+{{- if gt .cloudstackEtcdDiskOfferingSize 0}}
```

Comment:
Yes, it can work. CustomSize is an optional field for capc and eksa.

We may ends with customSize: 0 for non-customized disk offering which may looks weird. And if we don't provide it, its default value is 0, so technically, they both works.